Makefile

```diff
@@ -1,11 +1,11 @@
 PROG=bin/stolon-pgbouncer bin/stolon-pgbouncer-acceptance
 PROJECT=github.com/gocardless/stolon-pgbouncer
 VERSION=$(shell git rev-parse --short HEAD)-dev
-BUILD_COMMAND=GO111MODULE=on go build -ldflags "-X main.Version=$(VERSION)"
```

Comment:
We shouldn't use GO111MODULE when we're vendoring our dependencies, as it makes it easy to accidentally use non-vendored modules.